test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -98,27 +103,37 @@ class CommandValidateExecuteTest : public ::testing::Test {
             .copy());
   }
 
-  ExecutionResult validateAndExecute() {
-    auto executor = factory->getCommandExecutor(command);
-    if (executor->validate(*command, *wsv_query, creator->accountId())) {
-      return executor->execute(
-          *command, *wsv_query, *wsv_command, creator->accountId());
+  iroha::ExecutionResult validateAndExecute() {
+    shared_model::proto::Command new_command =
```

Comment:
I think you could rework it with shared_model command in test SetUp

Reply:
There is no builders for shared model commands